### ISSUE TITLE
[Suggestion] chunkSavestateUnpatch

### ISSUE THREAD

**Issue by DragonEggBedrockBreaking** [gnembon/carpet-extra#187]
Adds rule `chunkSavestateUnpatch` which, as the name suggests, re-lowers the limit of data required for a chunk savestate, as the new (4GB?) limit can (and will, source: experience) corrupt a world if a chunk has that much data.

**Comment by James103** [comment]
How exactly did you manage to fill a chunk to 4 GB of compressed data? Was it the same "write a bunch of uniquely random books, put them into shulker boxes, and place those shulker boxes (inside chests) in the chunk"? If not, what method did you use?

**Comment by DragonEggBedrockBreaking** [comment]
> How exactly did you manage to fill a chunk to 4 GB of compressed data? Was it the same "write a bunch of uniquely random books, put them into shulker boxes, and place those shulker boxes (inside chests) in the chunk"? If not, what method did you use?

Yes, but i had to use many mods books/shulkers/chests and longer books.

> As for the actual suggestion, maybe make the rule instead be `chunkSizeLimit`, with a default value of 128 MB, a maximum of 1024 MB at most (possibly 2032 or 2047 MB), and a minimum of 2 MB?

No. The default value should be 4gb, because that is vanilla behaviour, and carpet should not modify vanilla behaviour. Minimum value should be 2mb. when running the command, it should show you only those two options (to click on in chat), but then you can add any integer on the end of the command.
